scripts/package.js

```diff
@@ -9,6 +9,7 @@ const embed = require('./embed');
 
 const argv = process.argv.slice(0);
 const nodeVersion = argv[2];
+const staticBinary = /^true$/.test(argv[3]);
```

Comment:
this is getting a little ugly. At which point to we actually switch to parsing CLI arguments?

Reply:
agree...maybe next PR? :tongue: